GitHub.js

```diff
@@ -252,6 +262,28 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "web",
+		"url": "https://github.com/zotero/translators/blob/master/GitHub.js",
+		"items": [
+			{
+				"itemType": "computerProgram",
+				"title": "zotero/translators",
+				"creators": [],
+				"date": "2021-05-26T17:17:38Z",
+				"abstractNote": "Zotero Translators. Contribute to zotero/translators development by creating an account on GitHub.",
```

Comment:
While we're in here, can we strip the "Contribute to…" part (which may or may not be localized)? Really should only include the description entered by the repo owner.

Reply:
Sure, I'll give it a try.